release/cibuild.py

```diff
@@ -499,6 +508,16 @@ def upload():  # pragma: no cover
         ])
         subprocess.check_call(["docker", "push", be.docker_tag])
 
+    if be.should_upload_docker:
+        click.echo("Uploading ARMv7 Docker image to tag={}...".format(be.docker_tag + "ARMv7"))
+        subprocess.check_call([
```

Comment:
I think this is not needed - we are already logged in at this point.
You can combine the two `if`s as well.

Reply:
Updated to be in the same block.